core/src/ys/ys.clj

```diff
@@ -74,6 +74,16 @@
 (defn if [cond then else]
   (if cond then else))
 
+(defn cpp-compile-file [ys-file]
```

Comment:
I would call this `compile-ys-to-cpp` probably.

I might also make it take a string rather than a file.
The input could come from a file, string, url, database, etc...